subsys/bluetooth/controller/Kconfig

```diff
@@ -7,6 +7,56 @@
 
 comment "BLE Controller support"
 
+# The following symbols are enabled depending if the controller actually
+# supports the respective features. In case of Nordic SW LL, all of these are
```

Comment:
Your second sentence here is bound to be outdated. I would remove it. The information is maintained by the list of selects that the different LLL controller has.

Reply:
OK, will remove